examples/calls/fetch_call_log.bal

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/io;
+import ballerina/os;
+import ballerinax/twilio;
+
+// Account configurations
+configurable string accountSID = os:getEnv("ACCOUNT_SID");
+configurable string authToken = os:getEnv("AUTH_TOKEN");
+
+// This sample demonstrates a scenario where Twilio connector is used to fetch a call log.
+public function main() returns error? {
+
+    twilio:ConnectionConfig twilioConfig = {
+        auth: {
+            username: accountSID,
+            password: authToken
+        }
+    };
+
+    // Initialize Twilio Client
+    twilio:Client twilio = check new (twilioConfig);
+
+    // Call SID: An identifier of 34 digits in length that uniquely identifies a call (https://support.twilio.com/hc/en-us/articles/223180488-What-is-a-Call-SID-). 
+    // You can obtain this identifier by running the 'listCall()' or by accessing the TwilioConsole > Monitor > Logs > CallLogs
+    string CallSID = "CAeb8427d6e95108ff0a8953fa301d1f1f";
+
+    twilio:Call call = check twilio->fetchCall(CallSID);
+
+    io:print(`
+    Date Created: ${call?.date_created}
+    From Number: ${call?.'from}
+    To Number: ${call?.to}
+    Status: ${call?.status}
+    Start Time: ${call?.start_time}
+    End Time: ${call?.end_time}
+    Duration: ${call?.duration}S
+    Price: ${call?.price} ${call?.price_unit}`
```

Comment:
IMO it will look better if we indent these lines by one tab

Reply:
Addressed in [474f18f](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/474f18fdb6516b8ff29c450126cde2c618369527)